src/component/Navbar.js

```diff
@@ -1,12 +1,15 @@
 import './books.css';
 
 const Navbar = () => (
-  <nav className="">
+  <nav className="navbar">
     <ul className="nav-content">
       <li> BookStore CMS </li>
-      <li><a href="/">Books</a></li>
-      <li><a href="/Category">Categories</a></li>
+      <li><a href="/">BOOKS</a></li>
+      <li><a href="/Category">CATEGORIES</a></li>
```

Comment:
- [ ] [REQUIRED] Kindly apply the exact color, font alignment, and font family to the `BOOKS` and `Categories also`. Zeplin used `#121212` for the color and `Montserrat-Regular` as the font family but you used `#551A8B` and `Tahoma` as the color and the font family respectfully. Kindly fix this as it is a core requirement. I have attached n image of it ahead, kindly refer to that.

Reply:
- [ ] Great job on applying the changes to the logo, but kindly ensure the links also conform to the Zeplin design as the previous reviewer has requested. Right now they have the default link styling as you can see below.
![image](https://user-images.githubusercontent.com/90556221/180765683-abb17f5b-5bd0-4dff-a5b8-448ba1c7107c.png)